packages/svelteui-core/src/components/AspectRatio/AspectRatio.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+	import useStyles from './AspectRatio.styles';
+	import Box from '../Box/Box.svelte';
+	import type { AspectRatioProps as $$AspectRatioProps } from './AspectRatio';
+
+	interface $$Props extends $$AspectRatioProps {}
+
+	export let use: $$Props['use'] = [],
+		element: $$Props['element'] = undefined,
+		className: $$Props['className'] = '',
+		override: $$Props['override'] = {},
+		ratio: $$Props['ratio'] = 1;
+	export { className as class };
+
+	$: ({ cx, classes, getStyles } = useStyles(
+		{
+			ratio
+		},
+		{ name: 'AspectRatio' }
+	));
+</script>
+
+<!--
+@component
+
+Compose elements and components in a vertical flex container.
```

Comment:
nitpicking: can the be same description that it's given in the docs

Reply:
Good catch! thanks